### DISCUSSION TITLE
New script idea: syslog server

### DISCUSSION THREAD

**Discussion by jedis00** [tteck/Proxmox#2112] [Ideas]
I'd like a script to set up rsyslog to accept logging from other remote servers, such as my router.

**Comment by dmshimself** [comment]
Yes please and if a web browser to display/filter all the logs could be included, that would be the cherry on top

**Comment by Pyromane** [comment]
Graylog in the community version would be interesting here, for example

**Comment by panzer-arc** [comment]
https://tteck.github.io/Proxmox/#openobserve-lxc
